src/ngraph/pass/memory_visualize.cpp

```diff
@@ -246,25 +243,17 @@ int pass::MemoryVisualize::compute_op_weight(const shared_ptr<Node> exop)
     int mass = 0;
     // for input_decl in exop.input_decls:
```

Comment:
This looks like Python++.  I'm not seeing the value in keeping it around.  This should be removed.